elements/pfe-band/src/pfe-band.json

```diff
@@ -148,6 +148,11 @@
               }
             ]
           }
+        },
+        "grid-layout": {
+          "title": "Grid layout",
+          "type": "boolean",
```

Comment:
@starryeyez024 Could you add some documentation on this new attribute to the README?

Reply:
I might be missing something but it seems like you added this?
https://github.com/patternfly/patternfly-elements/commit/f950d94d661e7c7f4076fd57b7a88c429cb26d03

In my local copy of this branch (which is quite stale), this field doesn't exist.  ¯\_(ツ)_/¯  I only see a small change to the scss from me, and where I merged in master, back in November.

Reply:
I believe I added it in response to the  Sass adding the attribute. I can document etc but I'm away from keyboard until Sunday so it will have to wait.